research/notes.org

```diff
@@ -8,67 +7,95 @@ subject are linked.
 :EXPORT_FILE_NAME: spoofax-org-export
 :END:
 #+LATEX_CLASS: article-shifted
-Spoofax is an environment that allows for the completely /declarative/
-definition of a programming language. The definition is done using
+Spoofax is a platform that allows for the completely /declarative/
+definition of a programming language and accompanying IDE
+support\nbsp\cite{Kats10a}. Such a platform is called a /language
+workbench/. The definition of a programming language is done using
 high-level /meta-languages/ for each aspect of the programming
 language.
 
 To define a language declaratively means that one uses the
 meta-languages to specify /what/ the properties of a language are, and
```

Comment:
How shall we deal with these `, and` constructs this time? :p I have no strong feelings either way, as long as we are consistent.